README.md

```diff
@@ -58,7 +58,7 @@ In details, Dragonfly has the following features:
 
 ## Comparison
 
-For Dragonfly, no matter how many clients start the file downloading, the average downloading time is almost stable without increasement (12s in experiment, which means it only takes 12s in total for all client to finish downloading file/image).
+For Dragonfly, no matter how many clients start the file downloading, the average downloading time is almost stable without increase (12s in experiment, which means it only takes 12s in total for all client to finish downloading file/image).
```

Comment:
I think increment is better. WDYT?  @AllenZMC

Reply:
No, I think `increase` is better, see https://wikidiff.com/increment/increase